libmamba/include/mamba/api/configuration_impl.hpp

```diff
@@ -304,21 +304,21 @@ namespace YAML
     };
 
     template <>
-    struct convert<fs::u8path>
+    struct convert<mamba::fs::u8path>
```

Comment:
Why was using the canonical name necessary? We are already in the `mamba` namespace.

Reply:
No, here here are specializing in the `YAML::` namespace.